ciphey/basemods/Searchers/ausearch.py

```diff
@@ -205,9 +205,13 @@ def expand_crackers(self, node: Node) -> None:
 
         for i in self.get_crackers_for(type(res)):
             inst = self._config()(i)
+            info = inst.getInfo(res)
+            if info.success_likelihood < self.p_threshold:
```

Comment:
If we set `info.success_likelihood` to 0, will it skip it?

Reply:
Yes, unless the p_threshold parameter is set to zero (or negative i suppose), where it will try anyway. This means that users can opt in to try incredibly unlikely bs if they want

Reply:
How would a user set that? So II can update docs :)

Reply:
You would set it with `-p ausearch.p_threshold`. So for example this would set the threshold to 1: `ciphey -t "SGVsbG8gbXkgbmFtZSBpcyBiZWUgYW5kIEkgbGlrZSBkb2cgYW5kIGFwcGxlIGFuZCB0cmVl" -p ausearch.p_threshold=1`